build.yaml

```diff
@@ -446,9 +461,9 @@ common_cli:
     # make sure we're no longer in our virtualenv
     deactivate
     # install the new freshly baked package
-    brew update
-    brew tap ClusterHQ/tap
-    brew install \${RECIPE_FILE}
+    retry 'brew update'
+    retry 'brew tap ClusterHQ/tap'
+    retry 'brew install \${RECIPE_FILE}'
     brew test \${RECIPE_FILE}
```

Comment:
you need double quotes on L466,
RECIPE_FILE=CHOCOLATE_COOKIES
echo '${RECIPE_FILE}'
${RECIPE_FILE}
echo "${RECIPE_FILE}"
CHOCOLATE_COOKIES

bash won't expand variables when you use '